3p/integration.js

```diff
@@ -78,6 +79,8 @@ import {eplanning} from '../ads/eplanning';
  */
 const AMP_EMBED_ALLOWED = {
   taboola: true,
+  'mantis-recommend': true,
+  'mantis-video': true,
```

Comment:
Can you use the same embedding approach other video players use? This will be deprioritized like ads and there is no easy way to specify a poster image.